server/server.go

```diff
@@ -78,18 +77,18 @@ func logRequestHandler(writer http.ResponseWriter, request *http.Request, nextHa
 }
 
 // LogRequest - middleware for loging requests
-func (server HTTPServer) LogRequest(nextHandler http.Handler) http.Handler {
+func (server *HTTPServer) LogRequest(nextHandler http.Handler) http.Handler {
```

Comment:
is it a form of (preliminary) optimization?

Reply:
I needed to modify server instance in `Start()` https://github.com/Sergey1011010/insights-results-aggregator/blob/unit-tests/server/server.go#L165 to be able to shut it down later and changed every method according to style guides(should be either pointer or value for every method in struct)

Reply:
ah ok, I see. TY (for me, if receiver is pointer, it 'says' that the method is about to modify receiver, but dunno, let's use your approach)